client/task-list/tasks/products/products.js

```diff
@@ -30,10 +30,16 @@ const subTasks = [
 				<Pill>{ __( 'Recommended', 'woocommerce-admin' ) }</Pill>
 			</>
 		),
-		content: __(
-			'Use a template to add physical, digital, and variable products',
-			'woocommerce-admin'
-		),
+		content:
+			window.wcAdminFeatures && window.wcAdminFeatures.subscriptions
```

Comment:
Same logic addition as mention in the comment in the `product-template-modal`?

Reply:
I addressed this modification in the commit 906f0a9
Since there were a few tests that [will be removed soon](https://github.com/woocommerce/woocommerce-admin/pull/7736) failing, I deleted them.